recipe/meta.yaml

```diff
@@ -49,7 +49,7 @@ outputs:
         - requests-toolbelt >=0.9.1,<1
         - urllib3 >=1.26,<2
         # websockets
-        - websockets >=10,<11
+        - websockets >=10,<12
```

Comment:
It looks like while upstream's `master` has relaxed this pin, `3.4.1` still has the upper bound. 

https://github.com/graphql-python/gql/blob/v3.4.1/setup.py#L50-L53